src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -1602,30 +1602,27 @@ private void Button_MouseEnter(object sender, MouseEventArgs e)
             tb.Foreground = (Brush)bc.ConvertFrom("#cccccc");
             Image collapseIcon = (Image)sp.Children[1];
 
-            Uri imageUri;
-
-            // When hovered swap appropriate expand/collapse icons
+            // When hovered swap appropriate expand/collapse button state
             if (LibraryCollapsed)
-            { imageUri = new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/expand_hover.png"); }
-
-            else
-            { imageUri = new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/collapse_hover.png"); }
-
-            BitmapImage hover = new BitmapImage(imageUri);
-            collapseIcon.Source = hover;
+            {
+                Uri imageUri;
+                imageUri = new Uri(@"pack://application:,,,/DynamoCoreWpf;component/UI/Images/expand_hover.png");
+                BitmapImage hover = new BitmapImage(imageUri);
+                collapseIcon.Source = hover;
+            }
         }
 
         private bool libraryCollapsed;
 
-        // Retain last known library width prior to collapsing
-        public double LibraryWidthCache { get; set; } = 200;
+        // Default library width
+        private const int defaultLibraryWidth = 200;
 
         // Check if library is collapsed or expanded
         public bool LibraryCollapsed
         {
             get
             {
-                if (LibraryViewColumn.Width.Value < 2)
+                if (LibraryViewColumn.Width.Value < 20)
```

Comment:
@Racel If we want the button to appear sooner or later we can change the threshold here

Reply:
It is fine for now. Lets just merge